### ISSUE TITLE
Check Formatting not working

### ISSUE THREAD

**Issue by capcarr** [blacktop/atom-python-yapf#4]
It seems that the yapf (python) process hangs when checking the format. I was able to solve the problem by passing the --style parameter, as is done when calling formatCode (which should be done anyway?).
I'm running yapf version 0.2.0 with Python 2.7 under Windows 7.

**Comment by blacktop** [comment]
Hmm, I am using a fully updated Mac with python 2.7.10 and atom 1.0 and it isn't hanging.  I will try and get a Windows VM set up and test this there too.  So are you saying that if nothing is in the style setting (which means the style flag won't get set that causes the hang?

**Comment by capcarr** [comment]
Atom itself doesn't hang, but the python process does. Every time I call the check format from Atom, a new python process is spawned, which never returns and continually consumes CPU. I tried running yapf from the command line, the same happens. When I provide a style, it works. However, I tested on a Linux machine, and it works fine there. But shouldn't the style be also passed to  yapf when checking the format anyway?
